src/js/components/Stepper/Stepper.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import Step from '../Step/Step';
+import Row from '../Row/Row';
+
+type StepType = {
+  label: string,
+  value: number | string,
+};
+
+type Props = {
+  activeStep: number,
+  className?: string,
+  steps: Array<StepType>,
+};
+
+const Stepper = ({ className, steps, activeStep, ...other }: Props) => {
+  const classes = classNames(className, 'stepper');
+
+  if (!Array.isArray(steps) || !activeStep) {
+    return null;
+  }
+
+  return (
+    <Row {...other} className={classes}>
```

Comment:
I think it might be better to just use utility classes for layout because Row & Col bring with them extra styles e.g. this can have `d-flex flex-column flex-sm-row` which will behave the same as you currently have.